src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/CaseMapper.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.companieshouse.insolvency.delta.mapper;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Context;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.ValueMapping;
+import org.mapstruct.ValueMappings;
+import uk.gov.companieshouse.api.delta.CaseNumber;
+import uk.gov.companieshouse.api.insolvency.CaseDates;
+import uk.gov.companieshouse.api.insolvency.Links;
+import uk.gov.companieshouse.api.insolvency.ModelCase;
+
+@Mapper(componentModel = "spring", uses = {PractitionersMapper.class})
+public interface CaseMapper {
+
+    @Mapping(target = "dates", ignore = true) // mapped in AfterMapping
+    @Mapping(target = "links", ignore = true) // mapped in AfterMapping
+    @Mapping(target = "notes", ignore = true) // doesn't exist on source
+    @Mapping(target = "practitioners", source = "appointments")
+    @Mapping(target = "number", source = "caseNumber")
+    @Mapping(target = "type", source = "caseType")
+    @ValueMappings({
+            @ValueMapping(target = "MEMBERS_VOLUNTARY_LIQUIDATION", source =
+                    "MEMBERS_VOLUNTARY_LIQUIDATION"),
+            @ValueMapping(target = "CREDITORS_VOLUNTARY_LIQUIDATION", source =
+                    "CREDITORS_VOLUNTARY_LIQUIDATION"),
+            @ValueMapping(target = "COMPULSORY_LIQUIDATION", source = "COMPULSORY_LIQUIDATION"),
```

Comment:
We probably not need to mention if both source and target is same?

Reply:
We did it to be explicit - especially because of some values missing a corresponding mapping.